libvast_test/src/main.cpp

```diff
@@ -82,32 +78,41 @@ extern std::set<std::string> config;
 
 } // namespace vast::test
 
-int main(int argc, char** argv) {
+namespace {
+std::vector<std::string> get_test_args(int argc, const char* const* argv) {
```

Comment:
```suggestion

// Retrieves arguments after the '--' delimiter.
std::vector<std::string> get_test_args(int argc, const char* const* argv) {
```

Assuming my understanding is correct here.